src/resources/postcss/views/month/_multiday-event.pcss

```diff
@@ -50,21 +50,13 @@
 	}
 
 	.tribe-events-calendar-month__multiday-event--start {
-		border-top-left-radius: 6px;
-		border-bottom-left-radius: 6px;
-
-		.tribe-events-calendar-month__multiday-event-inner {
-			border-left: 2px solid var(--color-accent-primary);
-		}
+		border-top-left-radius: 19px;
```

Comment:
The height of our multiday events are 19px, so the radii should be half that height. We can also just use `50%` here.

Reply:
My first approach was to use 50% but it doesn't work, I'll go with 9.5

Reply:
Oh, given the length, yes 50% won't work. 9.5 is the way to go here.